core/src/test/java/hudson/util/CopyOnWriteListTest.java

```diff
@@ -50,20 +53,24 @@ public void serialization() {
         TestData td = new TestData();
 
         String out = xs.toXML(td);
-        assertEquals("empty lists", "<hudson.util.CopyOnWriteListTest_-TestData>"
-                + "<list1/><list2/></hudson.util.CopyOnWriteListTest_-TestData>",
-                out.replaceAll("\\s+", ""));
+        String expected = "<hudson.util.CopyOnWriteListTest_-TestData>"
+                + "<list1/><list2/></hudson.util.CopyOnWriteListTest_-TestData>";
+        assertThat(expected, CompareMatcher.isSimilarTo(out).ignoreWhitespace()
```

Comment:
To match the semantic of `assertThat`, actual and expected should be swapped (also in the other test).

Reply:
Nice catch. Fixed now